tests/cli/test_global_concurrency_limit.py

```diff
@@ -282,3 +285,200 @@ def test_disable_gcl_not_found():
         expected_output="Global concurrency limit 'not-found' not found.",
         expected_code=1,
     )
+
+
+@pytest.fixture
+async def global_concurrency_limit(session):
+    gcl_schema = ConcurrencyLimitV2(
+        name="test",
+        limit=1,
+        active_slots=1,
+        slot_decay_per_second=0.1,
+    )
+    model = await models.concurrency_limits_v2.create_concurrency_limit(
+        session=session, concurrency_limit=gcl_schema
+    )
+
+    await session.commit()
+    return model
```

Comment:
I had to initiate a real database session to verify that updating global concurrency limits was missing validation.